hatch/src/cli/commands/build.rs

```diff
@@ -17,57 +16,65 @@ impl<'build> Build {
       name: "build"
     }
   }
+
+  pub fn execute(&self, project: &Project) -> HatchResult<()> {
+    task::generate_assets(&project).with_context(|e| {
```

Comment:
Can we make this a method call?

Reply:
Make what a method call?

Reply:
I just noticed that this is called outside of the call to execute in the build command from the inside.  It should be on the outside as it is a separate task.

Reply:
I was referring to the duplicate calls to task::generate_assets with duplicate error messages.  In line with DRY we should make this a method that's callable in multiple places

Reply:
What is DRY?  I could bake it into the call of task::generate_assets.  We should just not add the context to calls to generate_assets.

Reply:
Dont Repeat Yourself